price-feeder/price-feeder.example.toml

```
@@ -1,4 +1,5 @@
 gas_adjustment = 1
+gas_price = "0.0001uume"
```

```suggestion
gas_price = "0uumee"
```
Setting default to zero (also `uume` -> `uumee`)